src/classes/PassageError.ts

```diff
@@ -23,17 +13,15 @@ export class PassageError extends Error {
      * @param {string} message friendly message,
      * @param {Error} err error from node-fetch request
      */
-    constructor(message: string, err?: FetchError) {
+    constructor(message: string, err?: ResponseError) {
         super();
 
-        this.message = message;
-
-        if (err && isFetchError(err)) {
-            this.statusCode = err?.code ? parseInt(err?.code) : 500;
-            this.statusText = err.name;
+        if (err) {
+            this.message = `${message}: ${err.message}`;
+            this.statusCode = 500;
             this.error = err.message;
         } else {
-            this.error = err?.message;
+            this.message = message;
```

Comment:
Changes were made to the errors we are returning. They provide more information but the info isn't always correct. Our function is `passage.user.activate` not `activateUser`. Looking for thoughts. Should we keep our old errors or provide slightly misleading errors with more info? @flanagankp 

Old Error Message:
`Could not activate user`

New error message:
`Could not activate user: Required parameter requestParameters.userId was null or undefined when calling activateUser.`